linode/granola-prod/config.nix

```diff
@@ -19,16 +19,10 @@ let
   secretsImportScript = pkgs.writeScript "secrets-import" ''
     #! /bin/sh
     set -eu
-    NETLIFY_AUTH_TOKEN="$(cat /run/keys/netlify-auth-token)"
-    export NETLIFY_AUTH_TOKEN
     CLOUDFLARE_ACCOUNT_ID="$(cat /run/keys/cloudflare-account-id)"
     export CLOUDFLARE_ACCOUNT_ID
     CLOUDFLARE_API_TOKEN="$(cat /run/keys/cloudflare-api-token)"
     export CLOUDFLARE_API_TOKEN
-    LINODE_OBJ_ACCESS_KEY="$(cat /run/keys/linode-access-key)"
```

Comment:
These keys should be left on the box because we may want to build (rollback) to a build that requires the Linode object store.